### DISCUSSION TITLE
Peer status is not displayed on iOS phones.

### DISCUSSION THREAD

**Discussion by vielang** [rustdesk/rustdesk#8658] [Q&A] [answered]
Peer status is not displayed on iOS phones. On Android phones, it displays normally. How can I fix it

![346466586-236a537a-6d56-41ae-a3a5-18db0cc3feaa](https://github.com/rustdesk/rustdesk/assets/66666649/5cbcc14a-461a-445e-9efe-df4ec693723e)

**Comment by rustdesk** [comment]
https://x.com/rustdesk/status/1804531445576220990

**Reply by vielang** [reply]
Its still not work

**Comment by tschettervictor** [comment, marked as answer]
Working in latest 1.3.0 Testflight version.
